pinax/stripe/models.py

```diff
@@ -197,10 +199,43 @@ class TransferChargeFee(models.Model):
     created_at = models.DateTimeField(default=timezone.now)
 
 
+class UserAccount(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             related_name="user_accounts",
+                             related_query_name="user_account",
+                             on_delete=models.CASCADE)
+    account = models.ForeignKey("pinax_stripe.Account",
+                                related_name="user_accounts",
+                                related_query_name="user_account",
+                                on_delete=models.CASCADE)
+    customer = models.ForeignKey("pinax_stripe.Customer",
+                                 related_name="user_accounts",
+                                 related_query_name="user_account",
+                                 on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("user", "account")
+
+    def clean(self):
+        if not self.customer.stripe_account == self.account:
+            raise ValidationError(_("customer.stripe_account must be account."))
+        return super(UserAccount, self).clean()
+
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        return super(UserAccount, self).save(*args, **kwargs)
+
+    def __repr__(self):
+        return "UserAccount(pk={self.pk!r}, user={self.user!r}, account={self.account!r}, customer={self.customer!r})".format(self=self)
```

Comment:
Nice!

Reply:
Once you taste `fstring`s it is hard to go back :wink:

Reply:
Gets a bit long with many (potentially long) fields though..